docs/notebooks/analysis/swept_langmuir/find_floating_potential.ipynb

```diff
@@ -0,0 +1,273 @@
+{
```

Comment:
<ul><li>processng -&gt; processing</li><li>"scanned for points that equal zero" - risky for floats? Maybe "up to a tolerance", if that's what happens there?</li><li>expoenetial -&gt; exponential</li></ul><p>I'm just glossing over whether the algorithm itself is sensible, though.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaPy/PlasmaPy/pull/889/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
>     * "scanned for points that equal zero" - risky for floats? Maybe "up to a tolerance", if that's what happens there?

In general I agree with you, but I don't think it necessary in this case.  Since the algorithm also collects point pairs that straddle `I=0`, those points that are really close to zero will still be collected.

Reply:
Oh, I get what you mean now! Okay, that's fine, then.